src/App.js

```diff
@@ -97,49 +96,83 @@ const checkLoginStatus = async () => {
 const teamRoles = {
   BACKEND_TEAM: 'backend team',
   FRONTEND_TEAM: 'frontend team',
+  MENTOR: 'mentor'
 };
-const { BACKEND_TEAM, FRONTEND_TEAM } = teamRoles;
+const { BACKEND_TEAM, FRONTEND_TEAM, MENTOR } = teamRoles;
 export const projectTeam = [
```

Comment:
I think this object would make more sense to live in the Team.jsx file. That way the component would be self-contained. If we had to move Team.jsx to a different app / project, it would be pretty easy if the data lived in Team.jsx.